docker-compose.yml

```diff
@@ -114,10 +114,31 @@ services:
     build:
       context: .
       dockerfile: cpp/Dockerfile
-    environment:
-      PARQUET_TEST_DATA: /arrow/cpp/submodules/parquet-testing/data
     volumes: *ubuntu-volumes
 
+  cpp-ci:
+    # Usage:
+    #   docker-compose build cpp-ci
+    #   docker-compose run cpp-ci
+    image: ursalab/arrow:cpp-ci
+    shm_size: 2G
+    cap_add:
+      # LeakSanitizer and gdb requires ptrace(2)
+      - SYS_PTRACE
+    build:
+      context: .
+      dockerfile: cpp/Dockerfile.ubuntu-bionic
+      args:
+        LLVM_VERSION: 7
+    environment:
+      ARROW_FLIGHT: "OFF"
```

Comment:
Is there a reason to turn this off?

Reply:
Because it was originally off in the travis build I migrated. I initially turned it off to speed up (the develop cycle with travis is excruciatingly slow) the build.

I can turn it back on again, it doesn't cut much build time, the bulk of the time is spent building protobuf & grpc, which is not triggered by this flag (but the protobuf_Source=BUNDLED...).

On this subject, we might investigate how to make grpc stop complaining about php and ruby versions. This would allow us to use the system packaged one.